git_cli_vs_gui/git_cli_vs_gui.md

```diff
@@ -0,0 +1,339 @@
+<!--
+module_id: git_cli_vs_gui
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 4.0.0
+language: en
+narrator: US English Female
+mode: Textbook
+title: Git Command Line Interface versus Graphical User Interface
+comment:  Compare the two ways of interacting with Git to decide which is best for you.
+long_description: There are two ways of using Git: (1) typing commands on a command line interface (CLI) or (2) clicking through options on a graphical user interface (GUI) software. This module will compare and contrast these two options to help you decide how to use Git in your work.
+estimated_time_in_minutes: 30
+
+@pre_reqs
+Learners should know what version control is and why it's important and should also understand what a command line interface is and how that's different from a graphical user interface.  
+
+To acquire basic knowledge of version control, we recommend our [Introduction to Version Control](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/git_intro/git_intro.md#1) module. 
+
+To acquire basic knowledge about command line interfaces, we recommend our [Demystifying the Command Line Interface](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_command_line/demystifying_command_line.md) module.  
```

Comment:
```suggestion
To acquire basic knowledge about command line interfaces, we recommend our [Demystifying the Command Line Interface](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_command_line/demystifying_command_line.md#1) module.  
```
I think it might be a good idea for us to start always adding the `#1` at the end of the module links-- then it always takes folks to the first page of a module, even if they have aggressive caching that might take them elsewhere (I've had this happen recently and been frustrated by it).

Reply:
Oh very good point!  I also find this stupidly annoying!